src/handlers/applicantsHandlers.ts

```diff
@@ -54,6 +54,50 @@ const getApplicants = (
     });
 };
 
+const getApplicantsByExternalId = (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+): void => {
+  db.sql('applicants.selectByExternalId', { externalId: req.query.externalId })
```

Comment:
It might be straightforward or might not. If I have a moment, I will try it out.

Reply:
I still haven't taken the time to get to this, maybe on an additional pass next week. Extracting a helper sounds easy but for some reason with middleware I think it's more tricky. Regardless, it is probably worth doing to avoid the duplication.

Reply:
Please do!

Reply:
Easy enough, it turns out.